packages/client/src/runtime/utils/printStack.ts

```diff
@@ -119,7 +119,8 @@ function parseStack({
         params.callsiteStr = ':'
       } else {
         // Why even all this effort? Because if a user calls the client instance "db", we want to be able to also say "db.user.findMany"
-        const prismaClientRegex = /(\S+(create|createMany|updateMany|deleteMany|update|delete|findMany|findUnique)\()/
+        const prismaClientRegex =
+          /(\S+(create|createMany|updateMany|deleteMany|update|delete|findMany|findUnique|findFirst|findUniqueOrThrow|findFirstOrThrow)\()/
```

Comment:
Wondering if we can build this regex dynamically from `allClientModelActions` array. Some of the actions are omitted here and i am not sure if it's intentional. 

On a tangent, this whole module might be a topic for deep dive, i feel

Reply:
I agree that this needs to be dynamically generated as this is unmaintainable to keep on updating!